Intersect.Server.Core/Entities/Entity.cs

```diff
@@ -2169,21 +2175,21 @@ public virtual void TryAttack(Entity target)
                 {
                     Animate(enemy, aliveAnimations);
                 }
-
-                //Check for any onhit damage bonus effects!
-                CheckForOnhitAttack(enemy, isAutoAttack);
             }
 
+            //Check for any onhit damage bonus effects!
+            CheckForOnhitAttack(enemy, isAutoAttack);
+
```

Comment:
Hello, gave this PR a try and it seems good so far, just wanted to know though, why was this one call changed outside the previous conditional branches?
as for the rest, tested with a pair of items, so far seems to behave as expected, approved 👍🏽